proposals/artifacts.md

```diff
@@ -0,0 +1,58 @@
+# OCI artifact project proposal
+
+## Abstract
+Container registries, implementing the [distribution-spec][distribution-spec], provide reliable, highly scalable, secured storage services for container images. Customers either use a cloud provider implementation, vendor implementations, or instance the open source implementation of distribution. They configure security and networking to assure the images in the registry are locked down and accessible by the resources required. Cloud providers and vendors often provide additional values atop their registry implementations from security to productivity features. 
+
+Applications and services typically require additional artifacts to deploy and manage, including [helm](https://helm.sh) for deployment and [Open Policy Agent (OPA)](https://github.com/open-policy-agent/opa/issues/1413) for policy enforcement.
+
+Utilizing the [manifest][image-manifest] and [index][image-index] definitions, new artifacts can be stored and served using the [distribution-spec][distribution-spec] without changing the actual distribution spec. This repository will provide  a reference for artifact authors and registry implementors for supporting new artifact types with the existing implementations of distribution.
+
+By providing an OCI artifact definition, the community can continue to innovate, focusing on new artifact types without having to build yet another storage solution (YASS). 
+
+## Proposal
+Under the http://github.com/opencontainers organization:
+
+- Create a new **artifacts** repository, named http://github.com/opencontainers/artifacts
+- Update [distribution][distribution-spec] to generically reference [manifest][image-manifest] and [index][image-index], with image as one of many artifact types it can store. 
+
+## Contents
+
+The repository will serve 2 primary goals:
+
+1. **artifact authors** - guidance for authoring new artifact types. Including a clearing house for well known artifact types.
+1. **registry operators and vendors** - guidance for how they can support new artifact types, including how they can opt-in or out of well known artifact types. 
```

Comment:
We should define more clearly what "support" means here. Clarify that opting-in is only accepting the media types in the manifests, the structure of the references do not change since only the manifest/indexes can define relationships between objects. Likewise opting-out is rejecting those media types. So for example supporting a new media type does not mean that the new types must be parsed for references or considered during garbage collection.

Reply:
Does that need to be claified as part of the TOB proposal or as part of the actual project (if/once approved)? 

_note that I agree with your general statment that clarification of what support means is important, just questioning if that's a topic for the project to rationalize or the TOB_

Reply:
Agreed. My assumption is this is detail that would be covered in the content submitted to the repo, once created. This PR is the creation of the repo. As we do PRs on the newly created repo, we'll iterate on specifics. The opt-in|out is an interesting topic I expect a bit of discussion. Including, do we need an API for discovering what media-types are supported for the given registry and repo.

Reply:
Maybe I was using too many words to just say, "make clear to operators that no action is required by accepting this proposal". So if you check media types today, you already have a whitelist, if you don't, then you are already accepting new artifact types.

Reply:
I can do that...